### PR TITLE
Low Code CDK: DpathExtractor extend

### PR DESCRIPTION
**Pull request by artem1205**
## What
Parse responses with nested lists (see tests)
Resolving https://github.com/airbytehq/airbyte/issues/20225

Complex response structure connectors:

1. [ChargeBee](https://github.com/airbytehq/airbyte/blob/96a2a18fb75bdd426aff265f253852fdf19a8e02/airbyte-integrations/connectors/source-chargebee/source_chargebee/components.py#L17-L57)
2. [Facebook Pages](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/components.py#L55-L97)
3. [Monday](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-monday/source_monday/dpath_string_extractor.py)

## How
Use dpath.util.values in DpathExtractor

## Recommended reading order
1. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

### CODE REVIEW

**Review by sherifnada — CHANGES_REQUESTED** [review]
Looks good! could you update the docs to describe how the `DpathExtractor` now works? specifically, we should tell users that `*` can be used to mean "for each element in the array"

**Review by girarda — APPROVED** [review]
🚢 🚢 🚢

File: docs/connector-development/config-based/understanding-the-yaml-file/record-selector.md [hunk 1]
```diff
@@ -152,6 +153,50 @@ The selected records will be
 ]
 ```
 
+### Selecting nested field
```
Comment (sherifnada) [inline docs/connector-development/config-based/understanding-the-yaml-file/record-selector.md:156]:
```suggestion
### Selecting fields nested in arrays 
```

**Review by sherifnada — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]
Nice work!

**Review by brianjlai — APPROVED** [review]
@artem1205 changes look good to me. However, we've modified the release process of the `airbyte-cdk` package a little bit. 

We're starting the process a bit. Instead of writing the changelog and bumping the version, we use the "Publish CDK Manually" Github action point it towards your branch, your commit hash, and including the minor bump w/ the changelog message. If you don't have access, we can trigger the workflow once you remove your code on the `CHANGELOG.md` which will push a commit w/ the version bumps to your current branch

File: docs/connector-development/config-based/understanding-the-yaml-file/record-selector.md (hunk 1 above)
Comment (artem1205) [reply docs/connector-development/config-based/understanding-the-yaml-file/record-selector.md]: updated

**Comment by artem1205** [comment]
@brianjlai , I've removed my code from the `CHANGELOG.md`.
As I understood, I need to merge this PR now, and then run the `Publish CDK Manually` Action?
image

**Pull request merged**